swarm/storage/feed/feed.go

```diff
@@ -58,7 +58,6 @@ func (f *Feed) binaryPut(serializedData []byte) error {
 	cursor += TopicLength
 
 	copy(serializedData[cursor:cursor+common.AddressLength], f.User[:])
-	cursor += common.AddressLength
```

Comment:
Even this increment is not needed, I believe that it is done to be obvious to move the cursor if the code changes in the future. I would suggest to add a short comment including the code that is removed.